netjsonconfig/backends/wireguard/schema.py

```diff
@@ -46,12 +46,91 @@
                         "pattern": "^[^\\s]*$",
                         "propertyOrder": 3,
                     },
+                    "dns": {
+                        "title": "DNS",
+                        "type": "array",
+                        "uniqueItems": True,
+                        "propertyOrder": 4,
+                        "items": {"title": "DNS", "type": "string"},
+                        "propertyOrder": 4,
+                    },
+                    "mtu": {
+                        "type": "integer",
+                        "title": "MTU",
+                        "minimum": 68,
+                        "propertyOrder": 5,
+                        "default": 1500,
+                    },
```

Comment:
@nemesifier I couldn't find a good way to allow integer fields with empty value. One option would be to allow both string and integer values: 

```json
{
  "oneOf": [
    {
      "type": "integer"
    },
    {
      "type": "string",
      "enum": [""]
    }
  ]
}
```

But this renders a select drop-down for choosing between string or integer. Thus, I went ahead with integer field with a default value.

Reply:
Probably we can set 1420 as default (check what's the safest default, for ipv6).

Reply:
Let's set 1280 as default but make sure to remove this default value if untouched, so that way Wireguard will select the best default automatically, then we can override it if needed.